docs/sources/mimir/configure/configure-shuffle-sharding/index.md

```diff
@@ -129,14 +129,27 @@ If you’re running a Grafana Mimir cluster with shuffle sharding disabled, and
 
 The shuffle sharding implementation in Grafana Mimir has a limitation that prevents you from abruptly decreasing the tenant shard size when shuffle sharding is enabled for distributors on the read path. This is because when shuffle sharding is disabled, the queriers check all ingesters for blocks, whereas when it's enabled, they only check the ones that are assigned to that tenant through the shuffle-shard.
 
-To safely decrease the tenant shard size, follow these steps.
+##### Method 1: Using read shard size override (recommended)
+
+For partition shuffle sharding in ingest storage architecture, you can use the `-ingest-storage.ingestion-partition-tenant-read-shard-size` configuration to gracefully decrease the tenant shard size without disabling shuffle sharding globally:
+
+1. Set the read shard size to the current write shard size value via `-ingest-storage.ingestion-partition-tenant-read-shard-size=<current-size>` (or per-tenant via `ingestion_partitions_tenant_read_shard_size` in runtime configuration).
+1. Decrease the write shard size via `-ingest-storage.ingestion-partition-tenant-shard-size=<new-smaller-size>`.
+1. Wait for at least the amount of time specified via `-blocks-storage.tsdb.retention-period`.
+1. Remove the read shard size override (set `-ingest-storage.ingestion-partition-tenant-read-shard-size=0` or remove the runtime config override).
+
+This method allows queries to continue reading from the previous larger set of partitions while new writes go to the smaller set, without requiring shuffle sharding to be disabled for all tenants during the migration.
+
+##### Method 2: Disabling shuffle sharding temporarily
+
+If read shard size override is not available (for classic ingester architecture or older versions), follow these steps:
```

Comment:
We shouldn't reference classic architecture here. This section pertains only to ingest storage architecture.

Also, earlier version of what? Mimir? These are the 3.0 docs, so presumably they are using 3.0.

I think we can just say `If read shard size override is not available . . . ` and leave it at that.